dmpr.py

```diff
@@ -248,45 +240,40 @@ def tick(self):
             self._recalculate_routing_table()
 
         now = self._get_time(priv_data=self._get_time_priv_data)
-        self.tracer.log(self.tracer.TICK, {'now' : now})
+        self.tracer.log(self.tracer.TICK, {'now': now})
         if now >= self._next_tx_time:
             self.tx_route_packet()
             self._calc_next_tx_time()
             self.transmitted_now = True
         else:
             self.transmitted_now = False
 
+    def stop(self):
+        self._reset()
+        now = self._get_time(priv_data=self._get_time_priv_data)
+        self.log.warning("stop DMPR core", time=now)
 
-    def stop(self, init=False):
```

Comment:
mhh, is this tested and checked in dmpr*d*?

Reply:
the init parameter was only used internally